src/registry.rs

```diff
@@ -58,7 +59,7 @@ impl RegistryCore {
             // the collector_id.
             if desc_id_set.insert(desc.id) {
                 // The set did not have this value present, true is returned.
-                collector_id += desc.id;
+                collector_id = (Wrapping(collector_id) + Wrapping(desc.id)).0;
```

Comment:
can we use https://doc.rust-lang.org/beta/std/primitive.i64.html#method.wrapping_add directly?